plugins/inputs/http_listener_v2/README.md

```diff
@@ -18,8 +18,12 @@ This is a sample configuration for the plugin.
   ## Address and port to host HTTP listener on
   service_address = ":8080"
 
-  ## Path to listen to.
-  # path = "/telegraf"
+  ## Paths to listen to.
+  # paths = ["/telegraf"]
+
+  ## Save path in path_tag
+  ## Do not include path in tag if path_tag is an empty string
+  # path_tag = ""
```

Comment:
This should be a boolean.

Reply:
My intention is to set `tag` key here, so no hardcoded tag name is being used

Reply:
To be honest, I think this is overengineered. You can still change the tag name with processors later and it makes the option harder to understand for the user. So my preference is to have a hard-coded tag-name.

Reply:
ok, but what if `path` is already included into metric which comes to the input?

Reply:
This can only happen if the user configured it in `http_header_tags` I think. If you really worry about it, add a check if it was set there.

Reply:
I'm concern about parsers. For example as I understand, prometheusremotewrite treats all labels as tags: https://github.com/influxdata/telegraf/blob/32d4234ae4edb39a9ae86915d457974d167a0d0c/plugins/parsers/prometheusremotewrite/parser.go#L37-L39

Reply:
Let me rephrase: Which value would you use for `path_tag` in your setup? Why isn't this a nice name for the tag then? :-)

There is always the risk of a collision between that and the "path-tag". I'm not bound to the name `path`, so we could use something more specific (`listener_path`?). But imaging you offer that option and set `path_tag="some_value_path"`. How do you guarantee that the data you receive does not contain that tag?

So my idea is to _detect_ if the name is already used in the parsed metric and then use e.g. `http_listener_v2_path` as tag-name instead or issue a warning that the name is already taken. This will cover the whole collision problem and you will need it even if you allow setting a custom name.

Reply:
Ok, I get it now :)
`path` is pretty popular name and there is high chance to be used already by some of the metrics. Could we just use `http_listener_v2_path`? It can be pretty complicated to detect if `path` is from original source or from `http_listener_v2`.

Due to above I would like to avoid `path` in my setup